docs/openapi3.yaml

```diff
@@ -1383,6 +1387,142 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error_response'
+  # Webhooks
+  /v1/webhooks:
+    get:
+      operationId: retrieve-webhooks
+      tags:
+        - Webhooks
+      summary: Retrieve webhooks
+      description: Retrieve all webhooks.
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+               schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/webhook'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+        '404':
+          description: Not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+    post:
+      operationId: create-a-webhook
+      tags:
+        - Webhooks
+      summary: Create a Webhook
+      description: Create a new Webhook.
+      responses:
+        '201':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/webhook'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+        '422':
```

Comment:
what is the use for 422?

Reply:
yeah, in a second thought it doesn't fit, removed it